activemodel/lib/active_model/translation.rb

```diff
@@ -62,11 +64,15 @@ def human_attribute_name(attribute, options = {})
 
       defaults << :"attributes.#{attribute}"
       defaults << options[:default] if options[:default]
-      defaults << MISSING_TRANSLATION
+      defaults << MISSING_TRANSLATION if !Translation.raise_on_missing_translations
+
+      raise_on_missing_translations = true if options[:raise].nil? && Translation.raise_on_missing_translations
```

Comment:
```suggestion
      raise_on_missing_translations = true if options.fetch(:raise, Translation.raise_on_missing_translations)
```

Reply:
Actually, you don't even need that, because `options` is splatted later. So you can just remove all that and call `.translate` this way:

```ruby
      translation = I18n.translate(defaults.shift, count: 1, raise: Translation.raise_on_missing_translations, **options, default: defaults)
```

Reply:
Aha! That's true :bulb: